keyboards/handwired/kolhoz/keyboard.json

```diff
@@ -0,0 +1,105 @@
+{
+    "manufacturer": "Vitali Fomine",
+    "keyboard_name": "handwired/kolhoz",
+    "maintainer": "vf1",
+    "development_board": "promicro_rp2040",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["GP14", "GP2", "GP6", "GP10", "GP27", "GP20", null, null],
+            ["GP15", "GP3", "GP7", "GP11", "GP28", "GP21", null, null],
+            [null, "GP4", "GP8", "GP12", "GP22", "GP18", null, null],
+            [null, "GP5", "GP9", "GP13", "GP26", "GP19", "GP16", "GP17"]
```

Comment:
```suggestion
            ["GP14", "GP2", "GP6", "GP10", "GP27", "GP20"],
            ["GP15", "GP3", "GP7", "GP11", "GP28", "GP21"],
            ["GP16", "GP4", "GP8", "GP12", "GP22", "GP18"],
            ["GP17", "GP5", "GP9", "GP13", "GP26", "GP19"]
```

Reply:
This is a good optimization for the diode matrix. In my version, I can determine the correspondence of the key to the pin by just the matrix array (not very convenient, but possible).
I would prefer to leave it as is. What do you think?